filetracker/cachecleaner.py

```diff
@@ -0,0 +1,199 @@
+import datetime
+import logging
+from argparse import ArgumentParser
+import time
+
+import filetracker
+
+
+logger = logging.getLogger('ftcachecleanerd')
```

Comment:
Wojtek powiedział, że fajnie by w logach mieć to samo, jak nazywa się komenda, ale w sumie 'filetracker.cachecleaner' IMO też wygląda ok.